app/src/main/res/drawable/thumb_faceit.xml

```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="900dp"
+    android:height="900dp"
+    android:viewportWidth="900"
+    android:viewportHeight="900">
```

Comment:
Please decrease size to at most 128x128

Reply:
Changed thumbnail size to 128x128 in ca70d29